src/Helper/DateTimeFormatter.php

```diff
@@ -148,6 +154,9 @@ protected static function createInstance(array $args = [])
 
     private function process(\IntlDateFormatter $formatter, \DateTimeInterface $date): string
     {
-        return $this->fixCharset($formatter->format($date->getTimestamp()));
+        $result = $formatter->format($date->getTimestamp());
```

Comment:
while looking at the code, why are we doing $date getTimestamp again when the IntlFormatter already can work with DateTimeInterface ? or is this a timezone thing?

Reply:
No idea, I was just setting a variable to check the return type, but didnt changed the code.

Reply:
yeah i didn't changed the code either so much when doing the DateTimeFormatter ;P